crates/starfish/core/src/block_header.rs

```diff
@@ -630,7 +641,7 @@ impl fmt::Debug for VerifiedBlockHeader {
 }
 
 /// VerifiedTransactions are transactions that correspond to an existing block
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
A custom Debug implementation would be better to have more control over what is printed - currently I think we'd print bytes of all the transactions + the serialized bytes which would spam the logs hard. Let's add a TODO to implement it.